scripts/test-setup.sh

```diff
@@ -12,6 +12,23 @@ CONT_NAME="${OS_TYPE}-${OS_VERSION}-${COMPONENT}"
 CERTGEN_REPO="https://github.com/redhat-qe-security/certgen"
 CERTGEN_PATH="openssl/Library/certgen"
 
+# Test sanity check
+# Check if all tests have rlGetTestState at their end
+FAILED_CHECKS=0
+FAILED_NAMES=()
+while read file; do
+    if ! grep -Pzoq "rlGetTestState.*$" "$file"; then
```

Comment:
why `-z` option to grep? isn't `-o` useless with `-q`?

I don't know the Perl regular expressions, but usually `$` specifies end of line, not end of file, and `.*` won't match newlines, so I'm not exactly sure that this makes sure that the `rlGetTestState` is "at their end"...